### ISSUE TITLE
Formatting not saved, tables are squashed by default

### ISSUE THREAD

**Issue by douglasg14b** [trey-wallis/obsidian-notion-like-tables#265]
### Describe the bug

Table settings are not saved, like autowidth and manually set widths. Each time a note is viewed again tables are squished.

### Steps to reproduce

1. Make a table
2. Put stuff in it
3. Change column widths
3. Close and reopen the file

### Expected behavior

1. Tables are full width by default like default tables
2. Column settings are saved.

### Are you using the mobile app?

No

### Obsidian debug info

```
SYSTEM INFO:
	Obsidian version: v0.15.9
	Installer version: v0.15.9
	Operating system: Windows 10 Pro 10.0.19044
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 1
	Restricted mode: off
	Plugins installed: 12
	Plugins enabled: 9
		1: Folder Note v0.7.3
		2: Editor Syntax Highlight v0.1.3
		3: Table of Contents v0.1.3
		4: Admonition v9.0.5
		5: Calendar v1.5.10
		6: Dictionary v2.21.1
		7: Pandoc Plugin v0.4.1
		8: Tasks v1.15.0
		9: Notion-Like Tables v4.3.1

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please first try updating your theme to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

```

**Comment by trey-wallis** [comment]
Hi @douglasg14b. This will be fixed in version 5.0.0

**Comment by trey-wallis** [comment]
@douglasg14b v5.0.0 just released. Please check it out! :)
